gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 05 18:01:21 BRT 2018
+#Wed Mar 28 11:58:05 BRT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Didn't catch this before, but what if we go with v4.6? Not something we need to do, only consider. Feel free to just skip over this comment haha

Reply:
Will do it. Then maybe we should already go to android 3.1.0

Reply:
Absolutely! Just let everyone know but yeah, totally for it.